app/src/main/res/layout/fragment_details.xml

```diff
@@ -26,7 +26,7 @@
                 android:layout_height="wrap_content"
                 android:layout_below="@id/txt1"
                 android:layout_alignStart="@id/end_txt"
-                android:layout_marginTop="50dp"
+                android:layout_marginTop="20dp"
```

Comment:
Any reason for changing the margin top?

Reply:
I was going to remove 20dp since it was added by mistake . I will correct it in the next commit.